### PR TITLE
Add code-elinka to the owners list

### PR DESCRIPTION
**Pull request by code-elinka**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -9,5 +9,6 @@ approvers:
 - panslava
 - cezarygerard
 - kl52752
+- code-elinka
```
Comment (aojea) [inline OWNERS:12]: do you use to skip the ladder and go to superpowers directly? don't you start as reviewer first?

Comment (code-elinka) [reply OWNERS]: @aojea valid comment! :crab: 
Check it now please

**Pull request merged**